lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php

```diff
@@ -22,17 +22,17 @@
   <?php print $styles; ?>
 
   <!--[if lte IE 8]>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/html5shiv/dist/html5shiv.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/es5-shim/es5-shim.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/es5-shim/es5-sham.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/respond.js/dest/respond.min.js'); ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/html5shiv/dist/html5shiv.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/es5-shim/es5-shim.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/es5-shim/es5-sham.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/respond.js/dest/respond.min.js' ?>"></script>
   <![endif]-->
```

Comment:
Personally, I feel like it's more readable to have the IE conditional comment in the markup rather than using the render array to generate it. It's certainly something we can change if there's a good argument for it.